packages/utils/src/loader/cdn-loader.ts

```diff
@@ -24,6 +25,22 @@ export class CdnLoader {
     }
   }
 
+  /**
+   * Strips any event attributes which could be used to
+   * maliciously hijack the application.
+   * @param element Element to check
+   * @returns {void}
+   */
+  private stripUnsafeAttributes (element: SVGElement): void {
+    const attributes = element.getAttributeNames();
+    for (const attribute of attributes) {
+      // Remove event attributes e.g., `onclick`
+      if (attribute.startsWith('on')) {
```

Comment:
`IE11` does not support `String.prototype.startsWith()`.

Reply:
It should come as part of the default polyfill and automatically added by babel